packages/client-sdk-nodejs/test/unit/integration/auth-client.test.ts

```diff
@@ -1,5 +1,15 @@
 import {InternalSuperUserPermissions} from '@gomomento/sdk-core/dist/src/internal/utils';
-import {AllDataReadWrite} from '@gomomento/sdk-core';
```

Comment:
I apologize for this comment, I realize it's not something that was introduced in this PR and that I must have just missed in an earlier PR, my bad.

Two observations about this file:

1. The path says `unit/integration`. it should be one or the other. It looks like this is just a unit test? if so can we move it up a directory?
2. It's pretty unusual for us to be writing tests that directly reference the generated GRPC classes. Is there a way that we could get the same test coverage that are after here using only our public API classes? Such a test would be far more valuable in my eyes.

I won't block merging the PR on those points but I'd really like to address #2 in a fast follow if possible, if we don't do it in this PR. I don't want people to start copy/pasting the pattern of writing tests that reference the generated classes unless it's truly necessary :)

But please tell me if I'm wrong and these tests have high value and cannot be written using only public API classes.

Reply:
1. I agree - it confused me too when I saw the tests. I will move it up.
2. These are essentially unit tests for testing the conversion from SDK types to the client proto types so they have to use the generated GRPC classes. The integration tests will be the high value tests since those will implicitly test the conversions as well. If there was an api that would describe an existing token, I could have written a test that created a token and described it back and verified the permissions were the same.

Reply:
okay i will defer to your judgment on #2.

Reply:
Moved the unit test out of the `unit/integration` folder.